.pipelines/.vsts-vhd-automation.yaml

```diff
@@ -69,7 +69,7 @@ stages:
     condition: eq('${{ parameters.CreateRelease }}', true)
     jobs:
     - job: Automation
-      timeoutInMinutes: 60
+      timeoutInMinutes: 0
```

Comment:
0?

Reply:
[The timeoutInMinutes allows a limit to be set for the job execution time. When not specified, the default is 60 minutes. When 0 is specified, the maximum limit is used (described above).](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/phases?view=azure-devops&tabs=yaml#timeouts)

Reply:
That will means it would run for up to 6 hours, right?